config/all-in-one/values.yaml

```diff
@@ -74,30 +71,41 @@ cp4waiops:
     # The namespace of CP4WAIOps.
     namespace: cp4waiops
 
-  # Specify whether or not to setup CP4WAIOps with sample integrations, e.g.:
-  # Humio, Kafka, Kubernetes, etc. after it is installed.
-  setup: true
+  # Setup CP4WAIOps after installed
+  setup:
+    # Specify whether or not to setup CP4WAIOps with sample integrations, e.g.:
+    # Humio, Kafka, Kubernetes, etc. after it is installed.
+    enabled: false
+    # Setup Humio integration
+    # humio:
+    #   enabled: false
+    # Setup Kafka integration
+    # kafka:
+    #   enabled: false
+    # Setup Kubernetes integration
+    # kubernetes:
+    #   enabled: false
 
 # -----------------------------------------------------------------------------
 # Robotshop parameters
 # -----------------------------------------------------------------------------
 robotshop:
 
   # Specify whether or not to install Robotshop.
-  enabled: true
+  enabled: false
 
 # -----------------------------------------------------------------------------
 # Humio parameters
 # -----------------------------------------------------------------------------
 humio:
 
   # Specify whether or not to install Humio.
-  enabled: true
+  enabled: false
 
 # -----------------------------------------------------------------------------
 # Istio parameters
 # -----------------------------------------------------------------------------
 istio:
```

Comment:
Can these parameters cp4waiops.setup, robotshop, humio and istio also be set as hidden parameters, then describe them in the doc ?

Reply:
Will be addressed in a separate PR